boot/freeldr/freeldr/arch/uefi/uefimem.c

```diff
@@ -0,0 +1,296 @@
+/*
+ * PROJECT:     Freeldr UEFI Extension
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     UEFI Memory Managment Functions
+ * COPYRIGHT:   Copyright 2022 Justin Miller <justinmiller100@gmail.com>
+ */
+
+/**** Includes ***********************************************************************************/
```

Comment:
Please use our usual presentation: line 80-chars long, and:
`/* INCLUDES ******* ....... */`
etc (for the other ones)